src/EventHub/EventHub.Autorest/README.md

```diff
@@ -146,6 +146,15 @@ directive:
     set:
       subject: PrivateLink
 
+  - where:
```

Comment:
Wrong Rename. Rename "KeyVaultProperty" to "Encryption" and "EncryptionRequireInfrastructureEncryption" to "RequireInfrastructureEncryption". @schaudhari6254888